e2e-tests/tests/tls-cert-manager/03-check-cluster.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - script: |-
```

Comment:
I don't think we should replicate users test here, see PXC test: https://github.com/percona/percona-xtradb-cluster-operator/blob/main/e2e-tests/tls-issue-cert-manager/run